library/mlops/exceptions/tracking.py

```diff
@@ -0,0 +1,6 @@
+
+def project_id_missing_exception():
+    return ValueError('Project ID is missing')
+
+def experiment_id_missing_exception():
+    return ValueError('Experiment ID is missing')
```

Comment:
To bo honest, experiment_id_missing_exception() is ambiguous. Firstly, experiment_id is None and secondly experiment with this id does not exist. In my opinion this should be split into two exceptions: experiment_id_not_specified and experiment_id_not_found.